src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/customer_threads.yml

```diff
@@ -4,7 +4,6 @@ admin_customer_threads:
   defaults:
     _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\CustomerThreadController::indexAction
     _legacy_controller: AdminCustomerThreads
-  #  _legacy_link: AdminCustomerThreads
```

Comment:
You shouldn't remove the legacy_link they are here to help the final step when we enable the page And no we can even relate them to a feature flag (but it should be added in a dedicated PR)